scripts/workers/submission_worker.py

```diff
@@ -249,6 +250,7 @@ def extract_challenge_data(challenge, phases):
             CHALLENGE_IMPORT_STRING.format(challenge_id=challenge.id)
         )
         EVALUATION_SCRIPTS[challenge.id] = challenge_module
+        return challenge_module
```

Comment:
Sorry but I don't think I understand exactly how returning the challenge module here will help solve the problem of loading multiple modules. Can you please elaborate more on how this is solving the problem?

Reply:
That was a temporary hack but I found a permanent fix for this while loading the modules dunamically: https://docs.python.org/3/library/importlib.html#importlib.invalidate_caches